torch/_dynamo/variables/base.py

```diff
@@ -187,14 +187,13 @@ def apply(
 
                 def update_object_dict(v):
                     changed = False
-                    rv = dict(v.__dict__)
+                    rv = v.__dict__
```

Comment:
We should audit all remaining uses of `apply()` to ensure this isn't doing any mutations we don't want.

I worry we have something like:
```
def visit(node):
   ...

# node result is ignored
VariableTracker.apply(visit, obj)
```

Previously that would not have any side effects.  Now it does.

Reply:
Yeah I've taken a look, all of the uses of apply either 1) eagerly realize all vars or 2) track live vars in side effects, so we are good here.